src/components/SVG/Delete.js

```diff
@@ -0,0 +1,22 @@
+const DeleteIcon = ({ options, setInitialData }) => {
+  const { width, height } = options;
```

Comment:
and why not stylize through the class? why is it necessary to pass parameters to svg?